src/styles/blocks/blocks.scss

```
@@ -0,0 +1,30 @@
+.page {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill,
+  #{"minmax(max(300px, (33.3%)), 1fr)"});
+}
+
+.container__box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  color: #fff;
```

You should create varible for this color, something like this👇
```suggestion
  color: $white-color;
```

---

Not necessary, it's used only once.